Exiled.API/Features/Map.cs

```diff
@@ -317,27 +317,6 @@ public static GameObject PlaceTantrum(Vector3 position)
             return tantrum.gameObject;
         }
 
-        /// <summary>
-        /// Spawns an amnestic cloud on the map.
-        /// </summary>
-        /// <param name="position">The position to spawn the amnestic cloud.</param>
-        /// <param name="owner">The owner of the cloud, optional.</param>
-        /// <returns>The cloud's <see cref="GameObject"/>.</returns>
-        public static GameObject PlaceAmnesticCloud(Vector3 position, Player owner = null)
```

Comment:
It doesn't work even with your changes, right?

Reply:
It doesn't work at all - the owner has to be 939, otherwise it'll error

Reply:
Ok